docs/en/advanced_tutorials/basedataset.md

```diff
@@ -1,3 +1,503 @@
 # BaseDataset
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/basedataset.html).
+## Introduction
+
+The Dataset Class in the algorithm toolbox is responsible for providing input data for the model during the training/testing process. The Dataset Class in each algorithm toolbox under OpenMMLab projects have some common characteristics and requirements, such as the need for efficient internal data storage format, supporting for the concatenation of differrnt datasets, dataset repeated sampling and so on.
+
+Therefore, **MMEngine** implements **BaseDataset Class** which provides some basic interfaces, and implements some DatasetWrappers providing the same interfaces. Most of the Dataset Class in the OpenMMLab algorithm toolbox meet the interface defined by the BaseDataset and use the same DatasetWrappers.
+
+The basic function of the BaseDataset is to load the dataset information. Here, we divide the dataset information into two categories. One is meta information, which represents the information related to the dataset itself and sometimes needs to be obtained by the model or other external components. For example, the meta information of the dataset generally includes the category information `classes` in image classification task, since the classification model usually needs to record the category information of the dataset. The other is data information, which defines the file path and corresponding label information of specific data info. In addition, another function of the BaseDataset is to continuously send data into the data pipeline for data preprocessing.
+
+### The standard data annotation file
+
+In order to unify the dataset interfaces of different tasks and facilitate multiple tasks training in one model, OpenMMLab formulate the **OpenMMLab 2.0 dataset Format specification**. dataset annotation files should conform to this specification, and the BaseDataset reads and parses data annotation files based on this specification. If the data annotation file provided by the user does not conform to the specified format, the user can choose to convert it to the specified format and use OpenMMLab's algorithm toolbox to conduct algorithm training and testing based on the converted data annotation file.
+
+The OpenMMLab 2.0 dataset format specification states that annotation files must be in the format of `json` or `yaml`, `yml` or `pickle`, `pkl`. The dictionary stored in the annotation file must contain two fields, `metainfo` and `data_list`. The `metainfo` is a dictionary containing meta information about the dataset. The `data_list` is a list in which each element is a dictionary and the dictionary defines a raw data info. Each raw data info contains one or more training/test samples.
```

Comment:
```suggestion
The OpenMMLab 2.0 dataset format specification states that annotation files must be in the format of `json` or `yaml`, `yml` or `pickle`, `pkl`. The dictionary stored in the annotation file must contain two fields, `metainfo` and `data_list`. The `metainfo` is a dictionary containing meta information about the dataset. The `data_list` is a list in which each element is a dictionary which defines raw data. Each data contains one or more training/test samples.
```

Reply:
too complex if we use to `which` in one sentence